utils/default_config.go

```diff
@@ -97,7 +97,8 @@ func createConfigFromFlags(ctx *cli.Context) *Config {
 		SnapshotDepth:          getFlagValue(ctx, SnapshotDepthFlag).(int),
 		StateDbSrc:             getFlagValue(ctx, StateDbSrcFlag).(string),
 		StateDbSrcDirectAccess: getFlagValue(ctx, SrcDbOverwriteFlag).(bool),
-		StateValidationMode:    EqualityCheck,
+		// TODO re-enable equality check once supported in Carmen
```

Comment:
nit: maybe create an issue for this todo?

Reply:
see #1121